PrivateHeaders/APPX/ZIP.h

```diff
@@ -327,6 +328,11 @@ namespace appx {
            << "xmlns=\"http://schemas.microsoft.com/appx/2010/blockmap\" "
            << "HashMethod=\"http://www.w3.org/2001/04/xmlenc#sha256\">";
         for (const ZIPFileEntry &entry : otherEntries) {
+            const std::string suffix = ".appx";
+            if (suffix.size() < entry.fileName.size() &&
+                    std::equal(suffix.rbegin(), suffix.rend(), entry.fileName.rbegin())) {
+                continue;
+            }
```

Comment:
Yes it (probably) does harm as it is excluded in the .appxbundle file produced by MakeAppx.exe on Windows. I remember Shoaib telling me that if we include it, the archive would not wouldn't work properly. I can double check but I would rather do what MakeAppx.exe does. Also, otherEntries variable in that function is zipFileEntries variable in WriteAppx function in APPX.xpp: it is a list of all zip entries that are basically used before and after the function call. Creating a new copy of that list that only excludes .appx files and is used only for this function doesn't seem like a perfect idea honestly. Are there any strong reasons to do this?